mergify_engine/actions/merge_base.py

```diff
@@ -102,6 +102,8 @@ def strict_merge_parameter(v):
 async def get_rule_checks_status(
     ctxt: context.Context,
     rule: typing.Union["rules.EvaluatedRule", "rules.EvaluatedQueueRule"],
+    *,
+    other_unmatch_conditions_return_failure: bool = True,
```

Comment:
```suggestion
    unmatched_conditions_return_failure: bool = True,
```
"other" does not mean much in this context 😢